eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -62,14 +62,22 @@ jobs:
     - ${{ if and(notin(variables['Build.Reason'], 'PullRequest'), eq(parameters.osGroup, 'windows')) }}:
       - HelixPerfUploadTokenValue: '$(PerfCommandUploadToken)'
     - ${{ if eq(parameters.runtimeType, 'wasm') }}:
+      # nodejs installation steps from https://github.com/nodesource/distributions
       - HelixPreCommandsWasmOnLinux: >-
           sudo apt-get -y remove nodejs &&
-          curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash - &&
+          sudo apt-get update &&
+          sudo apt-get install -y ca-certificates curl gnupg &&
+          sudo mkdir -p /etc/apt/keyrings &&
+          sudo rm -f /etc/apt/keyrings/nodesource.gpg &&
+          curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor --batch -o /etc/apt/keyrings/nodesource.gpg &&
+          export NODE_MAJOR=18 &&
+          echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list &&
+          sudo apt-get update &&
+          sudo apt-get install nodejs -y &&
           sudo apt-get -y install nodejs &&
```

Comment:
This is basically same line as the one above.

Reply:
Fixed